src/zap/zapinfo.cpp

```diff
@@ -3390,14 +3390,16 @@ unsigned ZapInfo::getClassNumInstanceFields(CORINFO_CLASS_HANDLE cls)
     return m_pEEJitInfo->getClassNumInstanceFields(cls);
 }
 
-
-CorInfoHelpFunc ZapInfo::getNewHelper(CORINFO_RESOLVED_TOKEN * pResolvedToken, CORINFO_METHOD_HANDLE callerHandle)
+CorInfoHelpFunc ZapInfo::getNewHelper(CORINFO_RESOLVED_TOKEN * pResolvedToken, CORINFO_METHOD_HANDLE callerHandle, bool * pHasSideEffects)
 {
-	if (IsReadyToRunCompilation())
-		return CORINFO_HELP_NEWFAST;
+	if (!IsReadyToRunCompilation())
+	{
+		classMustBeLoadedBeforeCodeIsRun(pResolvedToken->hClass);
```

Comment:
Nit: indentation

Reply:
Fixed. This method (as well as a few other methods) had tabs instead of spaces.